### ISSUE TITLE
Error sending request

### ISSUE THREAD

**Issue by Kis4r4gi** [KirovAir/TwilightBoxart#13]
hi, i receive this message when try download cover "Error sending request. Unable to connect to remote server A connection cannot be established as the destination computer expressly denied such a connection" greetings for your greatest tool...

**Comment by Erdnussflip007** [comment]
Yeah i get the same Error, but only on the newest version, the 0.6 version downloads cover arts just fine.
I'd like to have the bigger ones from 0.7 though.

**Comment by KirovAir** [comment]
That was my server being offline, sorry. It's back with the TwilightBoxart 2.0 release, so this should just work again: https://github.com/KirovAir/TwilightBoxart/releases/tag/2.0.0 Thanks for the kind words! :)
